superchain/extra/addresses/sepolia/op.json

```diff
@@ -10,8 +10,12 @@
   "ProxyAdminOwner": "0x1Eb2fFc903729a0F03966B917003800b145F56E2",
   "SystemConfigOwner": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
   "DisputeGameFactoryProxy": "0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1",
-  "AnchorStateRegistryProxy": "0xbCC6AB30a6A8aFb030112182c45D3cb64EF11952",
-  "DelayedWETHProxy": "0xCa36B00f885D83068f0CfFEe381F0c974de1d58e",
+  "AnchorStateRegistryProxy": "0x218CD9489199F321E1177b56385d333c5B598629",
+  "DelayedWETHProxy": "0xF3D833949133e4E4D3551343494b34079598EA5a",
```

Comment:
Interesting, why did the proxy addresses change?

Reply:
These are a bit weird to have a proxy for - they're actually behind the `FaultDisputeGame` implementation already and aren't accessed directly. So when we update `FaultDisputeGame` it points to a new proxy because we generally don't want to change the behaviour for existing games (which continue to use the old `FaultDisputeGame` for their lifetime).  These proxies really only exist to give us maximal flexibility if we need to intervene manually.

Reply:
I see, thanks